spec/features/admins/admin_deletes_a_user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature "Admin deletes a user", %q(
+As an admin
+I want to be able to delete a user
+So that I can clean up our database
+
+Acceptance Criteria
+[x] I must be able to delete a specific user from our service
+[x] All playlist associated with that user must be deleted as well
+[ ] All reviews associated with the user must be deleted
+) do
+
+  scenario "Admin deletes a user" do
+    admin_user = FactoryGirl.create(:user, admin: true)
+    non_admin_user = FactoryGirl.create(:review)
+
+    sign_in_as(admin_user)
+    visit admin_users_path
+    expect(page).to have_content non_admin_user.user.email
```

Comment:
There's nothing wrong with this line here, but it is kind of redundant and not related to what you are testing for. You are testing the specific atomic (as Dan would call it) feature of a user. You would write a separate test for checking that a page has content. Generally, if you see an expect(page) to have or not have content in the middle of a feature test, it probably means you can break up that test even further. Just get rid of this expect here and be cognizant of breaking down tests as far as possible.